root/etc/cont-init.d/30-config

```diff
@@ -12,8 +12,8 @@ HC_CONF[SITE_NAME]=${SITE_NAME:-healthchecks}
 HC_CONF[DEFAULT_FROM_EMAIL]=${DEFAULT_FROM_EMAIL:-noreply@example.com}
 HC_CONF[EMAIL_HOST]=${EMAIL_HOST:-smtp.gmail.com}
 HC_CONF[EMAIL_PORT]=${EMAIL_PORT:-587}
-HC_CONF[EMAIL_HOST_USER]=${EMAIL_HOST_USER:-username}
-HC_CONF[EMAIL_HOST_PASSWORD]=${EMAIL_HOST_PASSWORD:-password}
+HC_CONF[EMAIL_HOST_USER]=${EMAIL_HOST_USER-username}
```

Comment:
dont set a default here and it will end up being empty if the user doesnt set the env `EMAIL_HOST_USER`. this would be a better user flow.

Reply:
Agreed. Also what I'd prefer, but that will break with current implementation.
Waiting for recommendation from an official.

Reply:
I think you can set it to an empty value, and then below, check if that value is not empty and only include it in the config if it's not empty:

```
HC_CONF[EMAIL_HOST_USER]=${EMAIL_HOST_USER:-}
# ...
# ...
if [ ! -z "${HC_CONF[EMAIL_HOST_USER]}" ]; then
    # include in config here
fi
```

Reply:
Thanks @alex-phillips for your comment. I think we have two options to disable authentication
 1. provide `-e "EMAIL_HOST_USER="` (which is what this PR suggests)
 2. breaking change of default behaviour.

The question is not how to implement that, but rather if it's OK to change the default behaviour (go with option 2)? What's your take on that?